R/tm_missing_data.R

```diff
@@ -1173,19 +1184,19 @@ srv_missing_data <- function(id, datasets, dataname, plot_height, plot_width, gg
       options = list(language = list(zeroRecords = "No variable selected"), pageLength = input$levels_table_rows)
     )
 
-    teal.widgets::plot_with_settings_srv(
+    pws1 <- teal.widgets::plot_with_settings_srv(
```

Comment:
Here you nicely start form 1 with enumeration, we have to be consistent in other places.